.flake8

```diff
@@ -1,7 +1,11 @@
 [flake8]
 max-line-length = 88
-ignore = E501, E203, W503
-per-file-ignores = __init__.py:F401
+ignore = E501, E203, W503, ANN101, ANN102
+mypy-init-return = True
```

Comment:
Is this a mypy config setting? If so, please put it in pyproject.toml (see #199).

Reply:
Nope, it's a `flake8-annotation` setting. See here: https://pypi.org/project/flake8-annotations/

> Allow omission of a return type hint for `__init__` if at least one argument is annotated. See mypy's documentation for additional details.